train.py

```diff
@@ -481,12 +486,32 @@ def main(args):
 			onnxruntime.set_default_logger_severity(0)
 		(logits_, ) = onnxruntime_session.run(None, dict(x = waveform_input.cpu().numpy()))
 
+		wraper = models.OnnxModelWraper(args.onnx)
+		wraper_logits = wraper(waveform_input)['logits'][0]
+		args.fp16 = None
```

Comment:
this is probably the cause of the problems

Reply:
done